webboot/webboot.go

```diff
@@ -43,6 +44,7 @@ var (
 	ipv4     = flag.Bool("ipv4", true, "use IPV4")
 	ipv6     = flag.Bool("ipv6", true, "use IPV6")
 	dryrun   = flag.Bool("dryrun", false, "Do not do the kexec")
+	wifi     = flag.String("wifi", "GoogleGuest", "Wifi name")
```

Comment:
suggestion: 
Change 
"Wifi name"
 to
"ESSID"
it's something people will get right away
you can also change the flag name from
"wifi"
to 
"essid"

Reply:
Done.I didn't know WiFi was also called essid. Learnt something new today :)